index.html

```diff
@@ -249,9 +233,23 @@ <h2>Prepare for engaging during the meeting</h2>
      <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
      <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
      <!-- Computing etc. -->
-     <h2>Computing resources etc.</h2>
+     <h2>Computing resources, datasets etc.</h2>
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <!-- IN2P3 -->
+     <h3>IN2P3</h3>
+     <div class="item">
+        <div class="box">
+             <input type="checkbox">
+        </div>
+        <div class="text">
+            As an LSST DESC member, you can request the setup of an IN2P3 account, you will find all the necessary information <a href="https://doc.cc.in2p3.fr/en/Getting-started/start.html#computing-account">here</a>.
```

Comment:
https://doc.cc.in2p3.fr/en/Getting-started/start.html#computing-account is giving me a 404 Not Found error.

Reply:
done